cluster/test-network.sh

```diff
@@ -24,7 +24,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
-KUBE_CONFIG_FILE="config-default.sh"
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+export KUBE_CONFIG_FILE="config-default.sh"
```

Comment:
```
In ./cluster/test-network.sh line 28:
KUBE_CONFIG_FILE="config-default.sh"
^-- SC2034: KUBE_CONFIG_FILE appears unused. Verify it or export it.
```

Reply:
So previously `KUBE_CONFIG_FILE` hasn't been exported to the subprocess? Does this change the subprocess behavior?

Reply:
In the subprocess, the first call of `KUBE_CONFIG_FILE ` appears in the following statement:
hack/ginkgo-e2e.sh
```go
: ${KUBE_CONFIG_FILE:="config-test.sh"}
export KUBECTL KUBE_CONFIG_FILE
```
It checks and exports `KUBE_CONFIG_FILE`. 
So I think it is also OK to export `KUBE_CONFIG_FILE` here.

Reply:
so... we're setting KUBE_CONFIG_FILE to `config-default.sh` to that it doesn't get set to `config-test.sh`? why are we not using the test variant?

you may have uncovered an existing bug, or this might be introducing a new one... is there any way you can prove to me which value of KUBE_CONFIG_FILE is currently used in anything that runs this script?